chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/QuestionableActivityEmailJob.java

```diff
@@ -133,6 +134,14 @@ public void execute(final JobExecutionContext jobContext) throws JobExecutionExc
         LOGGER.info("********* Completed the Questionable Activity Email job. *********");
     }
 
+    private Integer getRangeInDays(final JobExecutionContext jobContext) {
+        Integer range = 7;
+        if (jobContext.getMergedJobDataMap().containsKey("range")) {
+            range = jobContext.getMergedJobDataMap().getInt("range");
+        }
+        return range * -1;
```

Comment:
There's no validation in here. I'm not sure if there should be, but we can't assume the UI will provide "valid" numbers. My guess is that we should have a min value and a max value, defined as environment.properties values, and ensure that whatever the end user passes in falls within that range, or is constrained to that range by the code, if we want to allow that

Reply:
Updated the code to allow a range of 1-365.